src/main/java/frc/robot/Robot.java

```diff
@@ -1,13 +1,25 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import frc.robot.commands.ShootOut;
+import frc.robot.commands.ShooterPID;
+import frc.robot.subsystems.ShooterSystem;
 
 public class Robot extends TimedRobot {
+    private XboxController xboxController;
+    private ShooterSystem shooterSystem;
+
+
 
     @Override
     public void robotInit() {
-
+        shooterSystem = new ShooterSystem(2000);
```

Comment:
```suggestion
        shooterSystem = new ShooterSystem();
```
since we deleted this parameter from the ShooterSystem we no longer have to put anything inside the Parenthesis.

Reply:
done